net-irc/weechat/weechat-4.0.4.recipe

```diff
@@ -54,7 +54,7 @@ BUILD_REQUIRES="
 	devel:libintl$secondaryArchSuffix
 	devel:liblua$secondaryArchSuffix >= 5.3
 	devel:libncurses$secondaryArchSuffix
-	devel:libpython3.9$secondaryArchSuffix
+	devel:libpython3.10$secondaryArchSuffix
```

Comment:
Seems like it should also have a `lib:libpython3.10$secondaryArchSuffix` in the REQUIRES too.  Does it actually link against `libpython`?

Reply:
I see it listed as an optional dependency for Arch Linux and I don't see libpython in the `ldd` output so I guess it's ok.  Haiku/Haikuporter could use a RECOMMENDS variable.

Reply:
My guess is that it will use what is available at runtime, same as for Geany.

Reply:
Right, but some of the optional dependencies are listed, like lua, but python isn't.

Reply:
Maybe not in their docs, in Arch they are listed as optional: https://archlinux.org/packages/extra/x86_64/weechat/